website/docs/styling-layout.md

```diff
@@ -29,7 +29,7 @@ Any CSS you write within that file will be available globally and can be referen
 
 ## Styling your site with Infima
 
-`@docusaurus/preset-classic` uses [Infima](https://infima-dev.netlify.com/) as the underlying styling framework. Infima provides flexible layout and common UI components styling suitable for content-centric websites (blogs, documentation, landing pages). For more details, check out the [Infima website](https://infima-dev.netlify.com/).
+`@docusaurus/preset-classic` uses [Infima](http://facebookincubator.github.io/infima/) as the underlying styling framework. Infima provides flexible layout and common UI components styling suitable for content-centric websites (blogs, documentation, landing pages). For more details, check out the [Infima website](http://facebookincubator.github.io/infima/).
```

Comment:
```suggestion
`@docusaurus/preset-classic` uses [Infima](https://facebookincubator.github.io/infima/) as the underlying styling framework. Infima provides flexible layout and common UI components styling suitable for content-centric websites (blogs, documentation, landing pages). For more details, check out the [Infima website](https://facebookincubator.github.io/infima/).
```

`http`?

Reply:
Yeah we plan to change that to https later. Surprisingly the build's URL is https o.o

See link here - https://v2.docusaurus.io/docs/styling-layout/#styling-your-site-with-infima

Reply:
It's strange but right :smile:

Reply:
I patched it here https://github.com/facebook/docusaurus/pull/1955